### ISSUE TITLE
Hyper-parameter to train seg model

### ISSUE THREAD

**Issue by seefun** [mit-han-lab/efficientvit#42]
Could you introduce the parameter settings for fine-tuning downstream tasks in details？ I am unable to reproduce the training results on cityscape when using efficientvit-b series models. And during trt deployment, fp16 may encounter nan or huge loss of accuracy.
